continuous_integration/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Let's make this directory name `CI` for brevity and to match the capitalization scheme of directories in this repo.

Reply:
done